faq.md

```diff
@@ -5,10 +5,15 @@ title: FAQ
 
 # FAQ
 
```

Comment:
This shouldn't be the first question. How the tables are structured is more important. They can scroll down to see how to contribute after that. In fact, this and the other question "Should everything be open-sourced?" could come right after each other at the end of table-related questions.

Reply:
I think it should because it's the first question that comes to mind when someone wants to contribute; and contribution is the main point of an open source project, and this one.  
I've asked a couple of people and they felt lost working with the website. And "updating" is not the same as "contributing" or "adding", which is what both the people mentioned and also pointed out that it's hard to find.  
Anyway, these are all suggestions and you don't have to agree with anything. I'm just saying it doesn't work like you think it should.

Reply:
I think the most important point of **any** project is the content. However, I understand the concern and think we can do something to make it easier but I don't think bringing up the sentence is the best way.
Regarding your second point, I didn't understand the point about updating and if we're still talking about "contribution" not being clear. Could you elaborate on that?

Reply:
Edit: I understood the point about updating vs contribution. However, the first part is still unclear to me.